server/routes/aws/cloudwatch_routes.py

```
@@ -0,0 +1,287 @@
+            "1. Open the AWS SNS console and create a new topic (Standard type)",
+            "2. Create a subscription: Protocol = HTTPS, Endpoint = the webhook URL above",
+            "3. After creating the subscription, Aurora will auto-confirm it",
+            "4. In CloudWatch, open each alarm → Actions → Add notification",
+            "5. Select the SNS topic you created for 'In alarm', 'OK', and 'Insufficient data'",
+            "6. Trigger a test alarm to verify the connection",
+        ],
+    })
+
+
+@cloudwatch_bp.route("/aws/cloudwatch/webhook/<user_id>", methods=["POST"])
+def alarm_webhook(user_id: str):
```

Ruff (0.15.14)

[warning] Too many return statements (8 > 6)

(PLR0911)